src/App/Fossa/Main.hs

```diff
@@ -107,6 +110,25 @@ experimentalLicenseScanCommand = command "experimental-license-scan" (info runIn
     runInit :: Parser (IO ())
     runInit = pure $ putStrLn "The 'experimental-license-scan' has been deprecated and renamed to 'license-scan'. Please use the 'license-scan' command instead."
 
+feedbackCommand :: Mod CommandFields (IO ())
+feedbackCommand = command "experimental-feedback" (info feedbackPrompt $ progDescDoc $ formatStringToDoc "Provide feedback on your FOSSA CLI experience, submit feature requests, and report bugs/issues")
```

Comment:
Talked with zach about running this command as an experiment. Is the process of doing so just having the command name tagged with "experimental"?

Reply:
Yes: https://github.com/fossas/fossa-cli/tree/master/docs/references/experimental#what-denotes-an-experimental-feature

Reply:
That being said, why should this be `experimental`? This isn't something we anticipate being scripted ever, right, so no need to worry about maintaining it forever. cc @zlav